### PR TITLE
Baltimore Map test

### PR DESCRIPTION
**Pull request by DeltaDav**
Adds the map Baltimore (WIP)

began with town

### CODE REVIEW

**Comment by DeltaDav** [comment]
At this state, the map is testable. Some area are closed, and are too be added later. Still need to have code it in, (but I got no idea how to do that, not a coder...)

Area closed : Navy ship Dungeon, Giant Cargo Ship dungeon, some station in the subway.

**Comment by DeltaDav** [comment]
OKAY ITS BACK

**Comment by xTrainx** [comment]
gotta change the Baltimore.json to baltimore.json

File: code/modules/jobs/job_types/map_changes/map_changes.dm [hunk 1]
```diff
@@ -1,4 +1,6 @@
 //this needs to come after the job_types subfolder to keep the correct ordering
 
 #include "..\..\..\..\_maps\map_files\Pahrump-AB\job_changes.dm"
+#include "..\..\..\..\_maps\map_files\Baltimore\job_changes.dm"
 #undef JOB_MODIFICATION_MAP_NAME
+
```
Comment (xTrainx) [inline code/modules/jobs/job_types/map_changes/map_changes.dm:6]: linters say this is missing a trailing newline, but i see it there, try adding an empty line with nothing in there at the end

File: tgui/public/tgui-panel.bundle.css
Comment (xTrainx) [inline tgui/public/tgui-panel.bundle.css:1]: Bruda restore this file you are gonna doom us all

File: code/modules/clothing/shoes/f13.dm [hunk 2]
```diff
@@ -331,3 +331,13 @@
 /obj/item/clothing/shoes/f13/tribal_sandals
 	icon_state = "tribaltreads"
 	icon_state = "tribaltreads"
+
+	//Minutemens
+
+	/obj/item/clothing/shoes/f13/minutemen
```
Comment (xTrainx) [inline code/modules/clothing/shoes/f13.dm:337]: Here be tab, please delete

**Review by xTrainx — APPROVED** [review]

**Review by xTrainx — APPROVED** [review]

**Review by xTrainx — APPROVED** [review]

**Pull request merged**